src/runtime/posix_error_handler.cpp

```diff
@@ -15,7 +15,7 @@ WEAK void halide_default_error(void *user_context, const char *msg) {
         dst[1] = 0;
         dst += 1;
     }
-    halide_msan_annotate_memory_is_initialized(user_context, buf, dst - buf + 1);
+    (void) halide_msan_annotate_memory_is_initialized(user_context, buf, dst - buf + 1);
```

Comment:
why the explicit discards of the return value?

Reply:
I think it's better to make it obvious that we know a value is returned but we are deliberately ignoring it.